### DISCUSSION TITLE
Attempting to update the script mod

### DISCUSSION THREAD

**Discussion by 4Drea** [TitanNano/TS4ControlAnySim#80] [Q&A] [answered]
Hi! I'm still learning coding and had a few questions regarding the 'Control Any Sim' mod. When opening the TS4ControlAnySim-master package into PyCharm and clicking the ControlAnySim.package, it claims to be loaded in the wrong endcoding : UTF-8. This is confusing me a bit - as I believe this is actually the correct endcoding?
Also, the code is absolutely unreadable (at least to me, unless I'm uaware of how to read and precieve what's being displayed. - Again, I'm still in school for coding so this might be the case.) If this isn't the case, I would like to know if this unreadable dispay is due to the 'wrong endcoding' error or, possibly, something else.
I know it's a lot, but if there's any advice in fixing this issue, I would love to know!

**Comment by TitanNano** [comment, marked as answer]
`ControlAnySim.package` is a binary package of all the assets the mod needs inside the game not the source code. This package mostly contains translations and interaction definitions. You can find copies of all the package contents inside the [`package`](https://github.com/TitanNano/TS4ControlAnySim/tree/master/package) folder.

The source code you are looking for is actually located under [`src/control_any_sim/`](https://github.com/TitanNano/TS4ControlAnySim/tree/master/src/control_any_sim).
